web/templates/compose_banner/long_paste_options.hbs

```diff
@@ -0,0 +1,14 @@
+<div
+  class="main-view-banner {{banner_type}} {{classname}}">
+    <div class="main-view-banner-elements-wrapper">
+        {{#if show_paste_button}}
+        <p class="banner_content">{{t 'Do you want to paste directly into the compose box, or convert the pasted text into a file?'}}</p>
+        <button class="main-view-banner-action-button paste-to-compose">{{t 'Paste'}}</button>
+        <button class="main-view-banner-action-button convert-to-file secondary-button">{{t 'Convert'}}</button>
+        {{else}}
+        <p class="banner_content">{{t 'Do you want to convert the pasted text into a file?'}}</p>
+        <button class="main-view-banner-action-button convert-to-file">{{t 'Yes, convert'}}</button>
```

Comment:
Looking at the `web/templates/compose_banner/compose_banner.hbs`, we should be adding a `right_edge` class to the right most element since this banner has a close button.